cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/tooling/DocBuilder.scala

```diff
@@ -110,6 +110,13 @@ trait DocBuilder {
     queryScope.addContent(new TablePlaceHolder(queryScope.assertions, queryScope.params:_*))
   }
 
+  def statsOnlyResultTable(): Unit = {
```

Comment:
We should use this for all the empty parentheses in security (at least) to make at least this part is consistent